src/js/angular/getImgService.js

```diff
@@ -0,0 +1,38 @@
+(function (angular) {
+    'use strict';
+
+    var app = angular.module('flickrPOC', []);
+    app.service('images', function ($http) {
```

Comment:
To keep the code clean and pre-formatted for Angular 2, please use the file structure described by John Papa in his styleguide.

https://github.com/johnpapa/angular-styleguide#services